platform/product-management/release-plan-template.md

```diff
@@ -6,6 +6,38 @@ So: **how might we craft a release plan to test our product "in the wild" at a s
 
 That's what this Release Plan Template is for! And note - there are feature toggles and beta banners at your disposal that you can use as a part of your plan.
 
+## Do I need unmoderated production testing?
+
+**Yes**, staged rollout is required unless you can confidently answer "yes" to all of the following:
+
+* This change does not add substantial new functionality to VA.gov
+* This change does not impact user flows through tasks
+* This change does not affect traffic to backend services
+
+*Example*: a change to a page's text content **could skip** staged rollout
+
+*Example*: a visual redesign to a page that doesn't affect user flows **could skip** staged rollout
+
+*Example*: adding a new field to an existing form, that doesn't **could skip** staged rollout
+
+*Example*: a new feature on an existing application that creates new backend traffic **needs staged rollout**
+
+*Example*: a significant change to how users navigate an existing form **needs staged rollout**
+
+*Example*: a feature that will route significantly more users (and therefore more backend traffic) to an existing application **neds staged rollout**
+
+### Exceptions
+
+If [feature toggles](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/tools/feature-toggles/) don't work for your use case, you can request an exception in Staging Review.
```

Comment:
Are we using feature toggles and staged release/phased rollout interchangeably? It's unclear here whether we're explaining how to get an exception for using feature toggles, and/or for doing a phased release. 

Maybe clarify like ... "If you determine that a phased release would be valuable to your development but feature toggles don't work for your use case, you can request an exception to the phased release requirement in your VSP Staging Review." ??? (Is that actually what we're trying to say?)

Reply:
made a small wording change here + went all-in on "staged rollout" wording, is this clearer?

Reply:
Somewhat? I still think it seems like we're using "feature toggles" and "staged rollout" interchangeably. Is that intentional? Are feature toggles our only tool for staged rollout? (I can think of something hacky — like, only redirect traffic from one legacy page but not another for a day ... but not sure we'd recommend that)

Reply:
> I still think it seems like we're using "feature toggles" and "staged rollout" interchangeably. Is that intentional?

I'm definitely trying not to --- staged rollout should refer to the process of rolling out to percentages of users at a time, feature toggles are the tool by which that's facilitated. Is there somewhere I've mixed those two up in here?

> Are feature toggles our only tool for staged rollout?

I _think_ so. I've added a sentence to state that explicitly that might help clear it up (right under "Exceptions"), and removed references to feature toggles elsewhere. IMO since this doc is more about the policy+process, this doc's wording should generally focus more on the "staged rollout" wording and less on implementation specifics (i.e. references to "feature toggles").